src/vm/index.md

```diff
@@ -105,7 +105,7 @@ For any input of type [`InputType.Coin`](../protocol/tx_format/index.md), a non-
 
 For each such input in the transaction, the VM is [initialized](#vm-initialization), then:
 
-1. `$pc`  and `$is` are set to the start of the input's `predicate` field.
+1. `$pc` and `$is` are set to the start of the input's `predicate` field.
```

Comment:
Maybe we should be more specific here around how gas factors into predicate verification? If we introduce the concept of `remaining_gas` here it would make the script execution update more clear.

Reply:
how should we ensure that we don't overly specify implementation for potentially parallelizing predicate execution? we eventually wouldn't be initializing predicate execution with remaining gas each time, just aggregating the resultant gas used.

Reply:
Good point, although even if we don't include remaining gas in initialization it should still be explicitly defined before using it for script initialization. Something like:

```math
remaining_gas =  \sum_{i=0}^p gas(predicates[i])
```